docs/plans/2026-02-27-product-card-actions.md

```
@@ -0,0 +1,449 @@
+1. Add import at top:
+```tsx
+import { ProductCardActions } from "@/components/storefront/product-card-actions";
+```
+
+2. After the closing `</div>` of the `{/* Content */}` div (line ~85), and before the closing `</Link>`, add:
+```tsx
+      <ProductCardActions product={product} />
+```
+
+Final structure inside `<Link>`:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)